docs/changes.txt

```diff
@@ -580,17 +580,61 @@ Major new features in this release
 
 
 
-3.0.4: (released 2017-??-??)
+3.0.4: (released 2017-03-08)
 ----------------------------
 
+All (non-GUI):
+
+- Don't crash on trailing '%' in wxDateTime::Format().
+- Fix various problems when parsing invalid ZIP files.
+- Fix generic wxTimePickerCtrl to accept max values from keyboard.
+- Multiple surrogate-related fixes in UTF-16 support.
+- Fix reading wide character data in wxFile::ReadAll().
+- Make parsing WAV data more robust.
+- Fix copy ctor in numeric validators classes.
+
 All (GUI):
 
 - Fix wxListbook::GetPageImage() to not always return -1 (Kinaou Hervé).
+- Don't reset scroll position to zero when window size changes.
+- Prevent use of deleted wxDocMDIChildFrame in wxView dtor.
+- Avoid delayed destruction if TLW was never created.
+- Unbind events when destroying wxTextEntryHintData.
+- Fix wxHTML parse bug in non-unicode, ? characters are not displayed.
```

Comment:
This is not very clear, I'd say something like

    - Fix display of non-breaking spaces in wxHTML non-Unicode build.